main/models.py

```diff
@@ -36,13 +36,17 @@ class Comment(models.Model):
     author = models.CharField(max_length=200)
     text = models.TextField()
     created_date = models.DateTimeField(default=timezone.now)
-    approved_comment = models.BooleanField(default=False)
+    is_approved = models.BooleanField(default=False)
 
     def approve(self):
         """Approve comment and save it in DB."""
-        self.approved_comment = True
+        self.is_approved = True
         self.save()
 
     def __str__(self):
         """Render Comment instance as its text by default when stringifying."""
         return self.text
+
+    def get_approved(self):
+        """Return comment if it is approved by the host."""
```

Comment:
Please DON'T write docstrings explaining a `return` statement. Write HIGH-LEVEL description about what the method really DOES.